notebooks/00_quick_start/sar_movielens.ipynb

```diff
@@ -115,14 +114,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 73,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [
```

Comment:
I think it is strange to include the notebook output sections. Though the ones in this notebook look clean, they will vanish as soon as the user executes the notebook. If it were put into Markdown before or after the code execution, then the user can see that their output is the same as the expected output.

Reply:
The updated outputs also make it a bit hard to follow the delta with this notebook in the code review

Reply:
the reason for having the outputs in the notebooks is because we want the notebooks to show the results. I can go to NCF notebook on github and see very quickly how it works, if we show no outputs then there is no much difference with a python file